src/me/hkgumbs/tictactoe/main/java/formatter/ThreeCharacterSlot.java

```diff
@@ -0,0 +1,21 @@
+package me.hkgumbs.tictactoe.main.java.formatter;
+
+import me.hkgumbs.tictactoe.main.java.board.Board;
+
+public class ThreeCharacterSlot implements SlotRepresentation {
+
+    @Override
+    public String compile(Board.Mark mark, int position) {
+        if (mark != null)
+            return " " + mark.toString() + " ";
+        else if (position < 10)
+            return "(" + Integer.toHexString(position) + ")";
+        else
+            return "...";
```

Comment:
If the string lengths are different, then it becomes impossible to center the board identically on every row. My other idea was to use hexadecimal, but that only hoists the problem up to 4x4. I couldn't think of a more general solution. Ideas?